docs/spec/consensus/fork-accountability.md

```diff
@@ -0,0 +1,142 @@
+# Fork accountability
+
+This document presents algorithms which are used to detect faulty validators in case of fork (so they can be 
+punished accordingly). Tendermint guarantees (agreement, validity and termination) hold only in case 
+faulty validators have at most 1/3 of voting power in the current validator set. In case this assumption
+does not hold, Tendermint agreement might be violated and we then have a fork. We say that fork is a case in
+which there are two commits for different block at the same height of the blockchain. Algorithms that we present
+in this document ensures that in those cases we are able to detect faulty validators (and not mistakenly accuse
+correct validators), and incentivize therefore validators to behave according to the protocol specification.
+
+## Fork scenarios
+
+There are several scenarios in which fork might happen:
+
+1. Equivocation: faulty validators sign multiple vote messages (prevote and/or precommit) for different values during
+   the same round r at a given height h. 
+
+2. Amnesia: faulty validators precommit some value v in round r (value v is locked in round r) and then prevote for different value v' in higher round r' > r without previously correctly unlocking value v. In this case faulty processes "forget" that they have locked value v and prevote some other value in the following rounds.
+
+3. Back to the past: faulty validators precommit some value v in round r from the past in which they haven't voted before. In this case some other value has been decided in a round r' > r, but there are some precommit messages signed by (correct) processes for value v in the round r. 
+
+4. Fantom validators: faulty validators vote (sign prevote and precommit messages) in heights in which they are not part of the validator sets (at the main chain).
+
+5. Lunatic validator: faulty validator that sign vote messages to support arbitrary application state. 
+
+## Types of victims
+
+We consider three types of potential attack victims: 
+
+- full node
+- lite client with sequential header verification (LCS) and 
+- lite client with bisection based header verification (LCB).
+
+In case of full node, attack surface is constrained to the blockchain as a correct full node executes 
+all transactions so it can verify if computed application state diverge from the proposed state. Therefore,
+full node can be tricked to follow forked chain by committing different set of transactions to a blockchain.
```

Comment:
I don't quite understand the final clause of this sentence - what is "tricked" and what is the "forked chain" in this case?